src/main/java/de/zalando/ep/zalenium/proxy/DockeredSeleniumStarter.java

```diff
@@ -56,6 +47,10 @@
     private static final Logger LOGGER = Logger.getLogger(DockeredSeleniumStarter.class.getName());
     private static final String DEFAULT_DOCKER_SELENIUM_IMAGE = "elgalu/selenium";
     private static final String ZALENIUM_SELENIUM_IMAGE_NAME = "ZALENIUM_SELENIUM_IMAGE_NAME";
+    public static final String DEFAULT_SELENIUM_CONTAINER_CPU_LIMIT = "100000000";
```

Comment:
Let's have the env vars name declared but without defaults. This part can all be done in the `DockerContainerClient.java` class. Basically, I think there is no need to touch this class either.  Can you please revert the changes here?